rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```
@@ -134,24 +144,49 @@ public Map<String, Object> bridgeState() throws IOException, BlockStoreException
     }
 
     public String call(CallArgumentsParam argsParam, BlockIdentifierParam bnOrId) {
+        return call(argsParam, bnOrId, Collections.emptyList());
+    }
+
+    public String call(CallArgumentsParam argsParam, BlockIdentifierParam bnOrId, List<AccountOverride> accountOverrideList) {
+
+        boolean shouldPerformStateOverride = accountOverrideList != null && !accountOverrideList.isEmpty();
+
+        if (shouldPerformStateOverride && !allowStateOverride) {
+            throw new UnsupportedOperationException("State Override is not supported");
```

maybe `throw invalidParamError("State override is not allowed");`?